security_controls_scp/modules/ec2/variables.tf

```
@@ -3,3 +3,18 @@ variable "target_id" {
   description = "The Root ID, Organizational Unit ID, or AWS Account ID to apply SCPs."
   type        = string
 }
+
+variable "AmiCreatorAccount" {
```

I'm thinking we can make this variable name [snake case](https://en.wikipedia.org/wiki/Snake_case), as is the de facto standard for Terraform. 
```suggestion
variable "ami_creator_account" {
```

---

Good Call, updated.